tests/test_converter.py

```diff
@@ -2107,8 +2122,8 @@ def test_convert_with_too_many_wires(self):
         a PennyLane operator with too many custom wires.
         """
         match = (
-            r"The specified number of wires - 3 - does not match "
```

Comment:
Might be better to keep the "r" for consistency with the rest.

Reply:
"r" is specifically used for matching string with regex, since there is no regex being used here, we don't really need to have this as a r-string.